flytekit/tools/fast_registration.py

```diff
@@ -32,6 +47,34 @@ class FastPackageOptions:
 
     ignores: list[Ignore]
     keep_default_ignores: bool = True
+    copy_style: Optional[CopyFileDetection] = None
+    ls_files: bool = False
+
+
+def print_ls_tree(source: os.PathLike, ls: typing.List[str]):
+    click.secho("Files to be copied for fast registration...", fg="bright_blue")
+    fff = []
```

Comment:
I do not see what `fff` means. Can this be renamed?

Reply:
debugging... removed.